### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by kirilloid**
Hi Fred,
I've though about the same idea and wanted to create my own solution, but then found out there's a solution already.
I've added several features and tried to split them into different commits.
If you think some changes are not suitable for the repository, please, look at separate commits and consider accepting some of them — I'll re-create pull requested with only picked ones.

### CODE REVIEW

File: mathml.css [hunk 1]
```diff
@@ -19,8 +19,15 @@ math[display="block"] {
...
-    vertical-align: -50%;
+    vertical-align: 30%;
     border-collapse: collapse;
+    line-height: 110%;
+    font-size: 90%;
+}
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    mfrac {
+        vertical-align: -50%;
+    }
```
Comment (Yaffle) [inline mathml.css]: The previous alignment was discussed at https://github.com/fred-wang/mathml.css/issues/2 .

**Comment by fred-wang** [comment]
Thanks for the pull request, I'll try to check it and give feedback ASAP.

File: mathml.css (hunk 1 above)
Comment (kirilloid) [inline mathml.css]: Well, these are valid points, but before my fix styles work different in Chrome and IE11 (I haven't checked MS Edge, though), so I made styles more consistent.

**Comment by fred-wang** [comment]
Sorry I don't have much time to focus on mathml.css (I prefer to focus on native mathml development!), but some quick comments:

0) Probably, it would be best to open different pull requests for the independent changes.

1) It seems that the WebKit blog examples have been modified to make them work better in mathml.css. I think we should not do that and instead be honest and show the limitation of this CSS stylesheet.

2) mspace.js is supposed to be loaded from an external URL as described in the README. Will that still work if we use a local URI of mspace.css?

3) What is the advantage of the mathvariant selectors (besides a simplified syntax)?  How do the performances compare? It seems that it will accept invalid attribute values...

4) In general, I don't like tweaking with arbitrary values as that works in some cases but not others...

**Comment by Zambonifofex** [comment]
Your changes seem to be inverting the sub and sup parts of the `msubsup` element [here](https://developer.mozilla.org/en-US/docs/Mozilla/MathML_Project/MathML_Torture_Test).

File: mathml.css (hunk 1 above)
Comment (Yaffle) [inline mathml.css]: @kirilloid , It seems, it will work for you, if you simply change `vertical-align: top;` to `vertical-align: text-top;`.

**Comment by fred-wang** [comment]
It seems there were disagreements on this PR. I honestly prefer to spend time focusing on native MathML than to review all this CSS stuff. Is anyone interested in helping with the maintenance of this repo and reviewing PRs?

**Comment by fred-wang** [comment]
Sorry, I just deleted master and used "gh-pages" as a reference and it seems this automatically closed your PR. feel free to submit a new one.

**Pull request closed without merging**